kotskinds/apis/kots/v1beta1/application_types.go

```diff
@@ -44,22 +44,23 @@ type ApplicationList struct {
 
 // ApplicationSpec defines the desired state of ApplicationSpec
 type ApplicationSpec struct {
-	Title                        string            `json:"title"`
-	Icon                         string            `json:"icon,omitempty"`
-	ApplicationPorts             []ApplicationPort `json:"ports,omitempty"`
-	ReleaseNotes                 string            `json:"releaseNotes,omitempty"`
-	AllowRollback                bool              `json:"allowRollback,omitempty"`
-	StatusInformers              []string          `json:"statusInformers,omitempty"`
-	Graphs                       []MetricGraph     `json:"graphs,omitempty"`
-	MinKotsVersion               string            `json:"minKotsVersion,omitempty"`
-	TargetKotsVersion            string            `json:"targetKotsVersion,omitempty"`
-	KubectlVersion               string            `json:"kubectlVersion,omitempty"`
-	KustomizeVersion             string            `json:"kustomizeVersion,omitempty"`
-	AdditionalImages             []string          `json:"additionalImages,omitempty"`
-	AdditionalNamespaces         []string          `json:"additionalNamespaces,omitempty"`
-	RequireMinimalRBACPrivileges bool              `json:"requireMinimalRBACPrivileges,omitempty"`
-	ProxyPublicImages            bool              `json:"proxyPublicImages,omitempty"`
-	ConsoleFeatureFlags          []string          `json:"consoleFeatureFlags,omitempty"`
+	Title                         string            `json:"title"`
+	Icon                          string            `json:"icon,omitempty"`
+	ApplicationPorts              []ApplicationPort `json:"ports,omitempty"`
+	ReleaseNotes                  string            `json:"releaseNotes,omitempty"`
+	AllowRollback                 bool              `json:"allowRollback,omitempty"`
+	StatusInformers               []string          `json:"statusInformers,omitempty"`
+	Graphs                        []MetricGraph     `json:"graphs,omitempty"`
+	MinKotsVersion                string            `json:"minKotsVersion,omitempty"`
+	TargetKotsVersion             string            `json:"targetKotsVersion,omitempty"`
+	KubectlVersion                string            `json:"kubectlVersion,omitempty"`
+	KustomizeVersion              string            `json:"kustomizeVersion,omitempty"`
+	AdditionalImages              []string          `json:"additionalImages,omitempty"`
+	AdditionalNamespaces          []string          `json:"additionalNamespaces,omitempty"`
+	RequireMinimalRBACPrivileges  bool              `json:"requireMinimalRBACPrivileges,omitempty"`
+	SupportsMinimalRBACPrivileges bool              `json:"supportsMinimalRBACPrivileges,omitempty"`
```

Comment:
to keep the naming consistent
```suggestion
	SupportMinimalRBACPrivileges bool              `json:"supportMinimalRBACPrivileges,omitempty"`
```